src/tests/unit/tests/reports/package/__snapshots__/integration.test.ts.snap

```diff
@@ -7223,7 +7219,7 @@ exports[`report package integration with no issues 1`] = `
       <div
         class="headerText"
       >
-        Accessibility Insights for Web
+        Accessibility Insights
```

Comment:
All other changes in this file are white-space related. I am not sure I understand why but that is why you see them.